converters/fromZigbee.js

```diff
@@ -331,7 +331,7 @@ const converters = {
         convert: (model, msg, publish, options, meta) => {
             const unlockCodes = [2, 9, 14];
             return {
-                state: unlockCodes.includes(msg.data['opereventcode']) ? 'UNLOCK' : 'LOCK',
+                command: unlockCodes.includes(msg.data['opereventcode']) ? 'UNLOCK' : 'LOCK',
```

Comment:
For my understanding, this event is send out whenever e.g. a lock command is executed? (regardless if it fails or not?).

For reference: these are the possible values.

![image](https://user-images.githubusercontent.com/2892853/82759146-b4896500-9deb-11ea-94a5-b922da7db514.png)

Reply:
Yes, this event is send out whenever a lock/unlock command is executed regardless of the result of the command. That is why I though if we want to keep the message (as it might be useful), it would be better to change the variable returned from `state` to `command` or something similar, as state contains the status of the lock, and updating this status before the lock responds with the real status would cause issues (i.e. false status if lock doesn't respond)

Reply:
Thanks, changed this to action and added a friendly text here so that it doesn't get cached.